clevertap-core/src/main/java/com/clevertap/android/sdk/CleverTapInstanceConfig.java

```diff
@@ -93,9 +73,20 @@ public CleverTapInstanceConfig createFromParcel(Parcel in) {
 
     private boolean useGoogleAdId;
 
+    public void setProfileKeys(@IdentityType String... profileKeys) {
+        if (!isDefaultInstance) {
```

Comment:
We don't need to check if it's a default instance because this method will be called before the instance creation. At that time, `isDefaultInstance` will be null